### ISSUE TITLE
Activity Stop Time is not accurate

### ISSUE THREAD

**Issue by davidtessier** [kevinpapst/kimai2#599]
**Issue**
When I use the Play / Stop button in the upper right hand of the application to stop an active task, the stop time seems to round the minute up for some reason. If the time is actually at 11:00 and I hit stop, it logs the end time as 11:01. If I then immediately start a new task, and it's truly still 11:00, then the new task starts at 11:00 and then there's an overlap with the previous task. The start time works fine, the stop time is the issue. This issue happens on just about every task I create.

**To Reproduce**
1. Start a new activity using the play button in the upper right. 
2. Wait at least a minute. (not sure if this matters)
3. Hit the stop button and note the time it says your activity ended at. 
4. Hit the start button right away to start a new task.
5. The stop time on the previous task should be a minute after it actually ended and the start time on your new task is the correct time. 

**Screenshots**
[Example screenshot](https://imgur.com/a/viWhGxS )

**Desktop/Smartphone**
 - Device: Lenovo Laptop 
 - OS: Windows 10 (64)
 - Browser Reproduced on Edge and Firefox
 - Microsoft Edge 42.17134.1.0 / Firefox 65.0.1

**Additional context**
 - Kimai version 0.8.1
 - The application is running on a Ubuntu 18.04 VPS with PHP 7.2.15 and is in the EST timezone.

**Comment by kevinpapst** [comment]
Hi @davidtessier ,
please read this: https://www.kimai.org/documentation/timesheet.html#rounding-of-begin-end-and-duration and configure Kimai to your needs.

By default start and stop time is rounded to full minutes, see this config:
https://github.com/kevinpapst/kimai2/blob/master/config/packages/kimai.yaml#L23

Man sure to read the configuration docu (do not edit kimai.yaml, but create local.yaml): 
https://www.kimai.org/documentation/configurations.html
Afterwards clear the cache and let me know if that solved your issue.

**Comment by davidtessier** [comment]
Thank you, @kevinpapst! Those links provided me with some great insight. As suggested, I created a local.yaml file, ran the commands to clear the cache and my changes are now in place. I did a test and noticed that the time is no longer rounding. Perfect! 

Side-bar...  
I've updated some of the photos for Kimai on https://alternativeto.net/software/kimai/ to better represent the look and feel of version 2. I am excited to see this system continue to grow. The more people that know about it the better!

Keep up the fantastic development. I really enjoy this tool.
